src/foam/dao/FileJournal.js

```diff
@@ -109,7 +110,23 @@ foam.CLASS({
     },
     {
       class: 'Boolean',
-      name: 'multiLine'
+      name: 'multiLineOutput',
+      value: true
+    },
+    {
+      class: 'Object',
+      name:  'multiLineOutputFiles',
+      javaType: 'java.util.HashSet',
+      javaFactory: `
+        HashSet<String> outputFiles = new HashSet<String>();
```

Comment:
I don't think we should do this.  First of all we're hard coding naonpay specific file locations in to the generic FOAM framework.  Secondly we already have the multilineOutput boolean property, just let client code set that property if they want to enable multiline output.